bundle.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=container-jfr-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.3.0
```

Comment:
Perhaps... I wanted to get some form of 1.x out, which ended up being 1.3, quickly, since rebasing or cherry-picking this against `main` has been a bit of a pain. I can attempt the 1.4.2 update in this branch as well and hope that it's a nice seamless upgrade. If not, then I would rather get 1.3 merged and do 1.4.2 soon after.

Reply:
Oh, no problem. I recognized something from the 1.4.0 migration guide, so I thought you had already updated.